Mesh_3/include/CGAL/Mesh_3/internal/parameters.h

```diff
@@ -0,0 +1,237 @@
+#ifndef CGAL_PARAMETERS_H
```

Comment:
You are missing the license header. You can copy/paste the first line from any file of that package.